api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptDao.java

```diff
@@ -0,0 +1,30 @@
+package org.pmiops.workbench.cdr.dao;
+import org.pmiops.workbench.cdr.model.Concept;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+
+public interface ConceptDao extends CrudRepository<Concept, Long> {
+
+    @Query(value = "select c.* from cdr.concept c where c.concept_name like %:conceptName% and c.domain_id in ('Condition','Observation','Procedure', 'Measurement', 'Drug') order by c.count_value desc limit 10;",
```

Comment:
Please break up this SQL into multiple lines; potentially define it as a constant somewhere and reference it here. (It's too hard to read this as-is.)

Reply:
Consider having these methods return Slice<Concept> or Page<Concept> instead of List<Concept>.  Or at any rate, making the limit value for the request a parameter (ideally passed up from the client), instead of hardcoding it in the SQL.

Reply:
You may want to use Pageable as the means for specifying ORDER BY and LIMIT, e.g.:

https://github.com/all-of-us/workbench/blob/26607d51cded481c0aa079b7e592cdb6963fbad2/api/src/main/java/org/pmiops/workbench/db/dao/ParticipantCohortStatusDao.java#L13

Reply:
Done breaking up. Paging coming later.